app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -325,13 +325,21 @@ export default class AgendaitemDecisionComponent extends Component {
 
   @action
   async updateBetreftContent() {
-    // *NOTE: approval decisions have a totally different text block.
-    // possible future work, for now we make sure the documents names are correct
-    const { shortTitle, title } = this.args.agendaContext.agendaitem;
-    const isApproval = this.args.agendaitem.isApproval;
+    const meetingKind = await this.args.agendaContext.meeting.kind;
+    const isMinisterraad = meetingKind.uri === CONSTANTS.MEETING_KINDS.MINISTERRAAD;
+    const { shortTitle, title, number } = this.args.agendaContext.agendaitem;
     const documents = this.pieces;
     const agendaActivity = await this.args.agendaitem.agendaActivity;
     const subcase = await agendaActivity?.subcase;
+    const titleStartsWithApproval = 
```

Comment:
@StuyckRuben We don't need all these extra checks anymore, just checking `this.args.agendaitem.isApproval` should be enough. 

This is different than in the ticket, but was discussed in this thread on chat: https://chat.semte.ch/group/kanselarij-dev?msg=HzebjCNLBosDJvGAN